inst/bind_pdf.R

```diff
@@ -1,15 +1,24 @@
-#' Binds together the PDFs from an array job - is called by 'pdf_array_job()'. 
+#' Binds together the PDFs from an array job - is called by 'pdf_array_job()'.
 #'
 bind_pdfs = function(temp_out_dir, final_out_dir, write_file_name){
   files = list.files(temp_out_dir, full.names=T, include.dirs=F)
   
+  # Add zeros to job ID so they will sort in the order of jobs file
+  order = data.table::data.table(file = files)
+  order[, order:=gsub(paste0(temp_out_dir, "/"), "", files)]
+  order[, order:=gsub(".png", "", order)]
+  order[nchar(order)==1, order:=paste0("000", order)]
+  order[nchar(order)==2, order:=paste0("00", order)]
+  order[nchar(order)==3, order:=paste0("0", order)]
```

Comment:
wait ... damn

from `?sprintf`
```
     ‘0’ For numbers, pad to the field width with leading zeros. For
          characters, this zero-pads on some platforms and is ignored
          on others.
```

and we're on "others" it seems

```
> o2
[1] "1"   "14"  "27"  "324"
> sprintf("%03s", o2)
[1] "  1" " 14" " 27" "324"
> sprintf("%3s", o2)
[1] "  1" " 14" " 27" "324"

Reply:
.... but then I find another way

```
> order = c("5", "1", "137", "14", "7")
> order
[1] "5"   "1"   "137" "14"  "7"  
> order[order(as.numeric(order))]
[1] "1"   "5"   "7"   "14"  "137"
```

Reply:
In conclusion, I think hard-coding the `nchar` is the wrong answer and in this case, after several exploratory methods, I think `order <- order[order(as.numeric(order))]` is **almost** the right answer.

I also think you shouldn't name it `order` since that's a built-in function. Even through R seems to be fine with it

Reply:
Oh duh... `as.numeric() ` is such an obvious fix. 🙄 And I'll change the variable name. Will be resolved in next commit.